### PR TITLE
Update _sensei.scss

### PR DESCRIPTION
**Pull request by cynthianorman**
background color change
PR for https://github.com/WordPress/Learn/issues/1874

### CODE REVIEW

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss [hunk 1]
```diff
@@ -144,3 +144,6 @@ details {
 		line-height: 2.8;
 	}
 }
+.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
+    background-color: #eee
```
Comment (krupal-panchal) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: Isn't this line ended with a semi-colon (;) in the last? For the consistent code practice according to the file.

Comment (cynthianorman) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: thanks - I've updated this

**Review by krupal-panchal — APPROVED** [review]

**Comment by jonathanbossenger** [comment]
@adamwoodnz when you have a moment, could you review this, and if you're happy with it, merge it in. Thanks

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss [hunk 2]
```diff
@@ -144,3 +144,6 @@ details {
 		line-height: 2.8;
 	}
 }
+.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
+    background-color: #eee;
+}
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: New lines above and below this please

```suggestion

.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
    background-color: #eee;
}

```

**Review by adamwoodnz — CHANGES_REQUESTED** [review]
Code changes look ok to me, other than the whitespace comment.

I can't reproduce this though, completion screens look like this for me:

Screenshot 2024-01-04 at 3 24 00 PM

Has the UI changed with a later Sensei version, or does it need to be a specific course?

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss (hunk 2 above)
Comment (cynthianorman) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: @adamwoodnz your screenshot doesn't show a module section divider in question here but rather just lesson links. So I'm wondering just like you if module names are somehow no longer available. I just accessed Sensei live to verify some courses I'm familiar with and I don't see data for modules.

Comment (adamwoodnz) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: @jonathanbossenger @bsanevans are you still able to reproduce this please?

**Comment by cynthianorman** [comment]
@adamwoodnz the module section divider is what we would like a different background colour for. 

Here's a screenshot of what's currently on the production site. Does this help?

Screenshot 2024-01-05 Course Complete Page

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss (hunk 2 above)
Comment (kaitohm) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: Yup, accessing this page below still shows the same as seen in the screenshot of #1874 
https://learn.wordpress.org/course-complete/?course_id=41529

Comment (adamwoodnz) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: Ok thanks @bsanevans, I still see this for that link, but I'll trust you both 🙂 

Screenshot 2024-01-08 at 3 15 34 PM

**Review by adamwoodnz — APPROVED** [review]
I can't reproduce this, but it's such a minor change I'm happy to ship it and @bsanevans and @cynthianorman can verify the fix.

**Pull request merged**